ui/components/component-library/popover/popover.tsx

```diff
@@ -37,7 +37,7 @@ export const Popover: PopoverComponent = React.forwardRef(
       referenceElement,
       isOpen,
       title,
-      isPortal = false,
+      isPortal = true,
```

Comment:
Shouldn't make any updates to the defaults if you need to change this prop do it at the implementation stage in `ui/components/multichain/global-menu/global-menu.js`